OTA.md

```diff
@@ -0,0 +1,118 @@
+# Over-The-Air (OTA) Updates
+
+
+**OTA is in active development. Breaking changes should be expected. Check this document frequently for updates.****
+
+
+## Workflow
+
+In [app.viam.com](app.viam.com), add the following to the `services` array; you can alternatively add a `generic` service then edit it to match the following
+
+### OTA Service Config
+
+```json
+{
+  "name": "OTA",
+  "namespace": "rdk",
+  "type": "generic",
+  "model": "ota_service",
+  "attributes": {
+    "url": <firmware-download-url>,
+    "version": <some-tag>
+  }
+}
+```
+
+
+In the `url` field, enter the url where a firmware will be downloaded from
+   - if using a local endpoint on the same network, remember to use the proper private ip address
+   - for cloud-hosted resources, embedded auth in the url is easiest, we don't support tokens yet
+
+
+The `version` field is equivalent to a `tag` and can be any arbitrary string of up to 128 characters. 
+After successfully applying the new firmware, this `version` will be stored in NVS. 
+This values is compared to that of the latest machine config from app.viam.com and will trigger the update process.
```

Comment:
`values` -> `value`? Also maybe clarify that a detected change in the version is what prompts the update process